docker/contract-verifier/Dockerfile

```diff
@@ -22,73 +22,19 @@ FROM debian:bookworm-slim
-    && mv zksolc-linux-amd64-musl-v1.3.5 /etc/zksolc-bin/v1.3.5/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.5/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.6 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.6 \
-    && mv zksolc-linux-amd64-musl-v1.3.6 /etc/zksolc-bin/v1.3.6/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.6/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.7 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.7 \
-    && mv zksolc-linux-amd64-musl-v1.3.7 /etc/zksolc-bin/v1.3.7/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.7/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.8 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.8 \
-    && mv zksolc-linux-amd64-musl-v1.3.8 /etc/zksolc-bin/v1.3.8/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.8/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.9 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.9 \
-    && mv zksolc-linux-amd64-musl-v1.3.9 /etc/zksolc-bin/v1.3.9/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.9/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.10 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.10 \
-    && mv zksolc-linux-amd64-musl-v1.3.10 /etc/zksolc-bin/v1.3.10/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.10/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.11 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.11 \
-    && mv zksolc-linux-amd64-musl-v1.3.11 /etc/zksolc-bin/v1.3.11/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.11/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.13 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.13 \
-    && mv zksolc-linux-amd64-musl-v1.3.13 /etc/zksolc-bin/v1.3.13/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.13/zksolc
-RUN mkdir -p /etc/zksolc-bin/v1.3.14 \
-    && wget https://github.com/matter-labs/zksolc-bin/raw/main/linux-amd64/zksolc-linux-amd64-musl-v1.3.14 \
-    && mv zksolc-linux-amd64-musl-v1.3.14 /etc/zksolc-bin/v1.3.14/zksolc \
-    && chmod +x /etc/zksolc-bin/v1.3.14/zksolc
-
-# install zkvyper
-RUN mkdir -p /etc/zkvyper-bin/v1.3.9 \
-    && wget https://github.com/matter-labs/zkvyper-bin/raw/main/linux-amd64/zkvyper-linux-amd64-musl-v1.3.9 \
-    && mv zkvyper-linux-amd64-musl-v1.3.9 /etc/zkvyper-bin/v1.3.9/zkvyper \
-    && chmod +x /etc/zkvyper-bin/v1.3.9/zkvyper
-RUN mkdir -p /etc/zkvyper-bin/v1.3.10 \
-    && wget https://github.com/matter-labs/zkvyper-bin/raw/main/linux-amd64/zkvyper-linux-amd64-musl-v1.3.10 \
-    && mv zkvyper-linux-amd64-musl-v1.3.10 /etc/zkvyper-bin/v1.3.10/zkvyper \
-    && chmod +x /etc/zkvyper-bin/v1.3.10/zkvyper
+# install zksolc 1.3.0-1.3.15
```

Comment:
1.3.12 should be skipped.. this zksolc was buggy and its binaries were removed

Reply:
done